src/cpp/session/SessionSuspend.cpp

```diff
@@ -433,6 +433,13 @@ void checkForSuspend(const boost::function<bool()>& allowSuspend)
       }
 
       // execute the forced suspend (does not return)
+      if (options().getBoolOverlayOption(kLauncherSessionOption))
+      {
+         // Avoid generating nonzero exit codes when running under Launcher.
+         // Error codes from normal behaviour like this are confusing for Slurm
+         // and Kubernetes administrators unfamiliar with our workloads.
+         suspendSession(true, EXIT_SUCCESS);
```

Comment:
A minor readability point... I'd rather see a variable `statusCode` that is set to EXIT_SUCCESS or EX_FORCE and passed to one suspendSession(..) call.  That important comment "does not return" will then apply to that single suspendSession call and the reader won't be left wondering if suspendSession would ever get called twice.

Reply:
Very sensible, and done. It makes the code in both places look identical, too.